tests/testAutopas/testingHelpers/GridGenerator.h

```diff
@@ -48,6 +48,37 @@ class GridGenerator {
                                 const Particle &defaultParticle = autopas::Particle(),
                                 const std::array<double, 3> &spacing = std::array<double, 3>{1, 1, 1},
                                 const std::array<double, 3> &offset = std::array<double, 3>{.5, .5, .5});
+  /**Fills Autopas Object with a Grid of Particles starting at
+   * @param autopas
+   * @param startingPositions
+   * @param particlesPerDim
+   * @param defaultParticle
+   * @param spacing
+   * @param offset
+   *
+   * */
+  template <class Particle, class ParticleCell>
+  static void fillWithParticlesOnR(autopas::AutoPas<Particle, ParticleCell> &autopas,
+                                   std::array<double, 3> startingPositions, std::array<size_t, 3> particlesPerDim,
+                                   const Particle &defaultParticle, std::array<double, 3> spacing,
+                                   std::array<double, 3> offset);
+
+  /**Fills Autopas Object with a Grid of Particles
+   * @param autopas
+   * @param startingPositions
+   * @param initialVelocity
+   * @param particlesPerDim
+   * @param defaultParticle
+   * @param spacing
+   * @param offset
+   *
+   * */
+  template <class Particle, class ParticleCell>
+  static void fillWithParticlesOnRInitialV(autopas::AutoPas<Particle, ParticleCell> &autopas,
```

Comment:
same as above applies. Also: Setting an initial `v` can already be achieved by setting it in the `dummParticle` before passing it to the function. This dummy is used as a copy-blueprint for all generated ones.